trace-dispatcher/src/Cardano/Logging/Tracer/Composed.hs

```diff
@@ -119,3 +126,32 @@ mkCardanoTracer' trStdout trForward mbTrEkg name namesFor severityFor privacyFor
         case mbEkgTrace <> mbForwardTrace <> mbStdoutTrace of
           Nothing -> pure $ Trace NT.nullTracer
           Just tr -> pure (preFormatted backends tr)
+
+-- A simple dataPointTracer which supports building a namespace and entering a hook
+-- function.
+mkDataPointTracer :: forall dp. ToJSON dp
+  => Trace IO DataPoint
+  -> (dp -> [Text])
+  -> IO (Trace IO dp)
+mkDataPointTracer trDataPoint namesFor = do
+    let tr = NT.contramap DataPoint trDataPoint
+    pure $ withNamesAppended namesFor tr
+
+documentTracer ::
+     TraceConfig
+  -> Trace IO a
+  -> Documented a
+  -> IO [(Namespace, DocuResult)]
+documentTracer trConfig trace trDoc = do
+    res <- catch (do
+              configureTracers trConfig trDoc [trace]
+              pure True)
+          (\(e :: SomeException) -> do
+            putStrLn $ "Configuration exception" <> show e <> show trDoc
```

Comment:
`putStrLn` isn't a good idea: I remember that Marcin asked me to use stdout tracer instead.

Reply:
Yes, `putStrLn` is one level lower than our minimum abstraction level for tracing -- `traceWith`.

Reply:
When this exception happens, the tracer is not usable. I can either rethrow the exception, but it is not shure what happens then. So I think priniting to stdout for debugging is acceptable in this case. Their is no possibility for this exception to be thrown in a running node, it is for developers.